src/storage/v2/replication/replication_client.cpp

```diff
@@ -305,12 +305,12 @@ void ReplicationStorageClient::RecoverReplica(uint64_t replica_commit, memgraph:
         std::visit(
             utils::Overloaded{
                 [&replica_commit, mem_storage, &rpcClient, main_uuid = main_uuid_](RecoverySnapshot const &snapshot) {
-                  spdlog::debug("Sending the latest snapshot file: {}", snapshot);
+                  spdlog::debug("Sending the latest snapshot file: {} to {}", snapshot, rpcClient.Endpoint());
```

Comment:
I think we mentioned we would log here also the replica name, right? Maybe not here but in the line 293 where we write "Starting replica recovery"

Reply:
I think any info is good. How can I fetch replica name?